### DISCUSSION TITLE
Best EPUB reader for Android, unavailable on stores?

### DISCUSSION THREAD

**Discussion by lavrentius-dev** [koreader/koreader#11722] [Q&A] [answered]
This app seems to be the most compliant free EPUB reader for Android. In my tests, it renders tables, lists, inline images, and embedded SVG content like the desktop Callibre reader. However, it is unavailable on the Google Play, Samsung, and F-Droid app stores. Anyone know why?

**Comment by benoit-pierre** [comment, marked as answer]
It is available on [F-Droid](https://f-droid.org/en/packages/org.koreader.launcher.fdroid/).

**Reply by lavrentius-dev** [reply]
![Screenshot_20240426_151332_F-Droid.png](https://github.com/koreader/koreader/assets/1596128/2b18f2c1-d64e-4556-ba1f-85d88f173e8a)

**Reply by benoit-pierre** [reply]
So you have an issue with your F-Droid? Why is it trying to connect to 127.0.0.1? Some kind of firewall is interfering? Notice you also don't have any screenshots. Additionally, the last KOReader update should be from 4 weeks ago.

**Reply by Frenzie** [reply]
It should look like this:
![Screenshot_2024-04-26-21-24-12-99_66a3b1c2c181ff92b8bf945e748b6fb7](https://github.com/koreader/koreader/assets/202757/7a2ce469-2cdc-46bc-a46f-1470f3103f6a)

**Reply by lavrentius-dev** [reply]
Ok, there were several technical problems on my end, all are solved now. I manually downloaded the latest F-Droid and was able to install KOReader. Thank you both!
